CMakeLists.txt

```diff
@@ -196,9 +195,6 @@ if(WITH_PIC OR ENABLE_SHARED)
     if(ENABLE_MEMKIND)
         target_include_directories(wt_objs PRIVATE ${HAVE_LIBMEMKIND_INCLUDES})
     endif()
-    if(ENABLE_TCMALLOC)
```

Comment:
If I'm understanding correctly, this may return in the future right? But with a very different implementation (i.e. fetching/building/linking against some specific version). Not a problem, just checking my understanding.

Reply:
Yes. How it is affected in CMake will be different, but we will explicitly link with a specific version of tcmalloc corresponding to one we built with the mongodb toolchain.